src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DecimalMathFuncs_NumberIsFloatDisabled_DVDecimal.txt

```diff
@@ -473,10 +473,10 @@ Error({Kind:ErrorKind.InvalidArgument})
 98765432198.12
 
 >> Round( 98765432198.1234567892, 100 )
-98765432198.1234567892
+Error({Kind:ErrorKind.Numeric})
```

Comment:
Do we need to make this an error? We can know exactly how many decimal places a decimal value has - see [https://stackoverflow.com/a/6092298/751090](https://stackoverflow.com/a/6092298/751090), so we can clamp the second argument to that value.